python/pyspark/logger/logger.py

```diff
@@ -20,6 +20,16 @@
 import json
 from typing import cast, Optional
 
+SPARK_LOG_SCHEMA = (
```

Comment:
Why did you move SPARK_LOG_SCHEMA from util.LogUtils to logger? SPARK_LOG_SCHEMA is in util.LogUtils for scala.

Reply:
There is no backward compatibility issue here. This is new in Spark 4. I think putting in the logger is more here is more appropriate。